apps/docs/src/main.tsx

```diff
@@ -1,11 +1,10 @@
 import { createRoot } from "react-dom/client";
-import "./style.css";
-import { Counter } from "@repo/ui";
+import { AmnCounter } from "@repo/design-system";
 
 const App = () => (
   <div>
     <div className="card">
-      <Counter />
+      <AmnCounter />
```

Comment:
I wouldn't worry about Prefixing all of your components with `Amn` - like Radix and others there isn't any point. Any consumers of your lib shouldn't care, a Switch is a Switch no matter where it came from.

If at any point someone has a conflict in naming, they can alias the import like `import { Switch as AmnSwitch } from ...` etc; but imagine this would be very edge case.